src/bootstrap.coffee

```diff
@@ -111,7 +111,10 @@ bootstrapper.startBootstrapping = ->
 		readConfigAndEnsureUUID()
 		.tap ->
 			loadPreloadedApps()
-		.tap ->
-			bootstrapOrRetry()
+		.tap (uuid) ->
+			if bootstrapper.offlineMode
+				return knex('config').insert({ key: 'uuid', value: uuid })
```

Comment:
No, `knex('config').select('value').where(key: 'uuid')` doesn't throw if the key isn't there. It just returns an empty array.